content/influxdb/cloud/reference/cli/influx/bucket-schema/list.md

```diff
@@ -0,0 +1,58 @@
+---
+title: influx bucket-schema list
+description: The `influx bucket-schema list` command lists measurement schema of a bucket in InfluxDB.
+menu:
+  influxdb_cloud_ref:
+    name: influx bucket-schema list
+    parent: influx bucket-schema
+weight: 201
+related:
+  - /influxdb/cloud/organizations/bucket-schema
+---
+
+The `influx bucket-schema list` command lists the measurement schema of a
+bucket in InfluxDB.
+
+## Usage
+
+```sh
+influx bucket-schema list --bucket <string>
```

Comment:
```suggestion
influx bucket-schema list [flags]
```

Reply:
> 
why is it different here? <bucket> is required.
`
jasonstirnaman@Jasons-MacBook-Pro influx-cli % ./bin/darwin/influx bucket-schema list
Error: bucket missing: specify bucket ID or bucket name
`